modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterManagementGroupManager.java

```diff
@@ -188,6 +193,8 @@ public void initCluster(
 
     @Override
     public void start() {
+        networkInvokeTimeoutMillis = configuration.networkInvokeTimeout().value();
```

Comment:
Do we want it to be dynamically-updated? If we do, then we should read it from the configuration every time

Reply:
We probably want it to be updated dynamically, switched to this style